R/metareactive.R

```diff
@@ -906,9 +730,17 @@ expandChain <- function(..., .expansionContext = newExpansionContext()) {
       # consider *themselves* their own dependencies, so for metaReactive
       # this means the code that assigns it is created (`mr <- ...`),
       # but the additional line for printing it (`mr`) will be suppressed.
-      x_vis <- withVisible(eval(as.symbol(paste0("..", i)), envir = environment()))
-      x <- x_vis$value
+      x_vis <- tryCatch({
+        withVisible(eval(as.symbol(paste0("..", i)), envir = environment()))
+      }, error = function(e) {
+        if (is_output_read(dot_args[[i]])) {
+          stop("Could not find: ", format(dot_args[[i]]), call. = FALSE)
```

Comment:
Hmm, I actually don't love this. Won't this throw an error even if `output$foo` does exist, but throws an error (either itself or one of its dependencies)? What's the error now, "attempt to apply non-function"?

Reply:
Yea, maybe I'll remove this for now and return to it later. I have a note of it in https://github.com/rstudio/shinymeta/issues/49